test/cypress/integration/lazy-chat.test.js

```diff
@@ -0,0 +1,46 @@
+import { crispWebsiteId } from '../../../src/client/static/data/app.json'
+
+const lazyChatButtonIdentity = '[data-test="lazyChatButtonIdentity"]'
```

Comment:
everything here is scoped to this file and we're already in the `lazy-chat` directory, so you can simplify your variable names to be more readable (`chatButton`, `optInDialog` (or whatever it is))